### ISSUE TITLE
installTwicPics fails with Svelte 5 & SvelteKit 2

### ISSUE THREAD

**Issue by Crenshinibon** [TwicPics/components#93]
I know Svelte 5 is currently still in beta. But I'm working on a Image and Video heavy project and plan to go with TwicPics and Svelte 5 right away.

On the basic setup I get the following error message, when running installTwicPics from root +layout.svelte file:

```
5:10:13 PM [vite] error while updating dependencies:
Error: Build failed with 5 errors:
node_modules/.pnpm/@twicpics+components@0.27.2/node_modules/@twicpics/components/sveltekit/TwicBackground.svelte:10:9: ERROR: No matching export in "node_modules/.pnpm/svelte@5.0.0-next.55/node_modules/svelte/src/internal/index.js" for import "get_current_component"
node_modules/.pnpm/@twicpics+components@0.27.2/node_modules/@twicpics/components/sveltekit/TwicImg.svelte:20:9: ERROR: No matching export in "node_modules/.pnpm/svelte@5.0.0-next.55/node_modules/svelte/src/internal/index.js" for import "get_current_component"
node_modules/.pnpm/@twicpics+components@0.27.2/node_modules/@twicpics/components/sveltekit/TwicPicture.svelte:31:9: ERROR: No matching export in "node_modules/.pnpm/svelte@5.0.0-next.55/node_modules/svelte/src/internal/index.js" for import "get_current_component"
node_modules/.pnpm/@twicpics+components@0.27.2/node_modules/@twicpics/components/sveltekit/TwicVideo.svelte:18:9: ERROR: No matching export in "node_modules/.pnpm/svelte@5.0.0-next.55/node_modules/svelte/src/internal/index.js" for import "get_current_component"
node_modules/.pnpm/@twicpics+components@0.27.2/node_modules/@twicpics/components/sveltekit/TwicView.svelte:9:9: ERROR: No matching export in "node_modules/.pnpm/svelte@5.0.0-next.55/node_modules/svelte/src/internal/index.js" for import "get_current_component"
    at failureErrorWithLog (/home/dirk/A/node_modules/.pnpm/esbuild@0.19.12/node_modules/esbuild/lib/main.js:1651:15)
    at /home/dirk/A/node_modules/.pnpm/esbuild@0.19.12/node_modules/esbuild/lib/main.js:1059:25
    at /home/dirk/A/node_modules/.pnpm/esbuild@0.19.12/node_modules/esbuild/lib/main.js:1527:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by mbgspcii** [comment]
Hi @Crenshinibon  and thanks for opening this issue.

Support for Svelte5 is planned and should be released soon.

I'll keep you informed as soon as a new compatible version is available.

**Comment by Crenshinibon** [comment]
Hi @mbgspcii 

thanks for the fast response and the good news. I'm looking forward for the Svelte 5 support.

**Comment by mbgspcii** [comment]
Hi @Crenshinibon 

We have just released version [0.28.0](https://www.npmjs.com/package/@twicpics/components/v/0.28.0) which should fix your problem.

This version has been tested with `Sveltekit` based on `Svelte5` **5.0.0-next.69** and **5.0.0-next.70**

It has also been tested with a pure `Svelte5` project.

Of course, as Svelte5 is still under development, it's not out of the question that new issues may arise.

Please keep us informed.

**Comment by Crenshinibon** [comment]
@mbgspcii Thank you very much. I will test it later today and I'll let you know if I encounter any issues.
